docs/rbd-nbd.md

```diff
@@ -28,6 +29,41 @@ client-side, which is inside the `csi-rbdplugin` node plugin.
 To use the rbd-nbd mounter for RBD-backed PVs, set `mounter` to `rbd-nbd`
 in the StorageClass.
 
+### Configuring logging path
+
+If you are using the default rbd nodeplugin daemonset and StorageClass
+templates then `cephLogDir` will be `/var/log/ceph`, this directory will be
+a host-path and the default log file path will be
+`/var/log/ceph/rbd-nbd-<volID>.log`. rbd-nbd creates a log file per volume
+under the `cephLogDir` path on NodeStage(map) and removed the same on
+the respective NodeUnstage(unmap).
```

Comment:
@pkalever just to understand, I think the main reason for this patch set was that, somehow we were looking for rbd nbd logs to be persisted. if `nodestage` and `nodeunstage` create and remove the log dir, I am not sure how that helps?  if the plan is to get the logs while staged, why we need hostdir? Am i missing something here?

Reply:
This was according to the discussion in our previous standup call.

Yes, we clean the logs in `unmap` and not on `detach`. Why do we want to analyze the logs of successful volumes? Mainly we will be looking into the logs when we see some issue with the volume and those logs will not be removed as there will not be a unmap on the odd ones or fail to map in the first place.

The idea here is to be more space-efficient.